src/Directory.build.targets

```diff
@@ -53,4 +53,7 @@
   <PropertyGroup Condition="$(TargetFramework.StartsWith('tizen'))">
     <DefineConstants>$(DefineConstants);TIZEN</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="$(TargetFramework.Equals('net6.0')) or $(TargetFramework.Equals('net5.0')) or $(TargetFramework.Equals('netstandard2.1')) or $(TargetFramework.Equals('netstandard2.0'))">
```

Comment:
We no longer have net5 support so can be removed from the condition

Reply:
Should I make another PR removeing NET5.0 altogether? Since you say splat doesnt support it, but .csprojs says otherwise